src/main/java/com/conveyal/r5/analyst/cluster/PathWriter.java

```diff
@@ -33,22 +32,26 @@
 
     OutputStream outputStream;
 
-    public PathWriter (AnalysisTask task, TransportNetwork network, List<Path> pathList) {
+    public PathWriter (AnalysisTask task, TransportNetwork network, List<Path> pathList, int destinations, int iterations) {
         this.task = task;
         this.network = network;
         this.pathList = pathList;
         try {
             outputStream = MultipointDataStore.getOutputStream(task, task.taskId + "_paths.dat", "application/octet-stream");
             outputStream.write("PATHGRID".getBytes());
+            // In the header store the number of destinations stored and iterations used
+            writeInt(destinations);
```

Comment:
Rather than calling our own method I think we should just wrap the OutputStream in a LittleEndianDataOutputStream to write full-width ints to it. Probably 75% of this file will be zeros but we will worry about that only if it actually proves to be a problem. I'm wondering whether we should just change MultipointDataStore.getOutputStream to always return a stream that has LittleEndianDataOutput capabilities.